### DISCUSSION TITLE
LTSP Image Kernel parameters change

### DISCUSSION THREAD

**Discussion by tichzvidzayi** [ltsp/ltsp#239] [Q&A]
(SERVER MACHINE) Dell Intel(R) Xeon(R) CPU E3-1225 v5 @ 3.30GHz

(THICK CLIENT)Mecer Xtreme Jupiter Mini PC  4GB DDR4-2400, (2 x 288PIN DDR4 DIMM Slot, Upgradable to 32GB)

Hi All,

When I install Mate or Ubuntu as the LTSP image my client machine runs into an infinite loop of PCI Bus Error severity = corrected, And when I google it the suggestion is to change kernel PCIe parameters but how do you change kernel parameters of the image. Or does anyone has another solution for the problem?

Regards 

--------------------------------------------------------------------------------------

If the issue you're about to report is likely to result in LTSP code or documentation changes, file an LTSP issue instead:
https://github.com/ltsp/ltsp/issues

**Comment by alkisg** [comment]
See the KERNEL_PARAMETERS option in the [ltsp.conf man page](https://ltsp.org/man/ltsp.conf/).

Btw, do delete the "If the issue you're about to report...." paragraph when filing new issues, don't keep it...
